science/seqan3/Portfile

```diff
@@ -28,6 +28,9 @@ github.tarball_from   releases
 distname              ${github.project}-${github.version}-Source
 use_xz                yes
 
+depends_run           port:bzip2 \
```

Comment:
I suspect these dependencies are also needed at configure/build time. If that’s indeed the case you should use “depends_lib-append” (using append is preferred anyway as not to accidentally overwrite dependencies that might be added by portgroups).

Reply:
Thank you for the explanation, I changed it accordingly.